src/llm/openai.rs

```diff
@@ -1,13 +1,15 @@
 use std::fmt::Display;
 
 use crate::{
-    llm::LLM,
+    llm::{EMBEDDING, LLM},
     prompt::{chat::Message, Prompt},
```

Comment:
I added this identifier as it was getting confused with the Embedding Struct, happy to change it if you have a better idea!